### DISCUSSION TITLE
mkv output

### DISCUSSION THREAD

**Discussion by cisko99za** [ErrorFlynn/ytdlp-interface#131] [General]
hi mate, very thanks for this tool, really amazing
only 2 question:
1) why no mkv output ? really pop today, maybe even more so than mp4
2) from ytdlp-interface page you wrote: 32 bit build (works on Win7). also 64bit version can work on win7. i've tested. so your tool work for everyone! thanks again

**Comment by ErrorFlynn** [comment]
Version 2.12 added `mkv` container preference in the settings, be sure to update!

**Comment by cisko99za** [comment]
yep, i can see! wonderful
